.goreleaser.yml

```
@@ -0,0 +1,45 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod download
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - id: grpcserver
+    binary: grpcserver
```

Can you change this for consistency with the package name? Can you also update the `Makefile` to use the package name for the server binary?
```suggestion
    binary: aws-encryption-provider
```